resources/definitions/inat_proton_x_rail.def.json

```diff
@@ -0,0 +1,31 @@
+{
+
+    "name": "Proton X Rail",
+    "version": 2,
+    "inherits": "inat_base",
+    "metadata": {
+        "visible": true,
+        "quality_definition": "inat_base",
+        "platform": "inat_proton_x_buildplate.stl",
+        "platform_offset": [ 0, -0.8, 0]
+    },
+    "overrides": {
+        "machine_name": { "default_value": "Proton X Rail" },
+        "machine_width": { "default_value": 304 },
+        "machine_depth": { "default_value": 304 },
+        "machine_height": { "default_value": 675 },
```

Comment:
The height of the build volume is 675. I'm guessing that would be to almost match the height of the largest edition of your printer, the 650mm version? I would be in favour of creating definitions that inherit from these definitions that then reduce the height! That would provide an easier printing experience, less likely to cause damage due to user error. I'll accept it as-is because you're not required to submit profiles for all your printers. But consider adding some height variations in the future :smiley: